about_us.md

```diff
@@ -3,4 +3,149 @@ layout: about_us
 title: About Us
 permalink: /about/
 exclude_from_nav: true
----
\ No newline at end of file
+current_members:
```

Comment:
this is amazing, i always hated updating the old list since it often even required css changes if the number of people changed